lib/kitchen/directions/bake_injected_exercise/bake_injected_exercise_question.rb

```diff
@@ -16,8 +16,8 @@ def self.v1(question:, number:, options: {
   class V1
     def bake(question:, number:, options:)
       id = question.id
-
       in_appendix = question.has_ancestor?(:page) && question.ancestor(:page).has_class?('appendix')
+      alphabetical_multipart = question.search('.alphabetical-multipart')&.first&.present?
```

Comment:
a good habit to be in with `#search` is to include, if possible, the `name` of the element as well as stuff like classes, data-types, etc. since i believe this should always be a div, this would become `question.search('div.alphabetical-multipart')` (or `question.search('div[data-type="alphabetical-question-multipart"]'` to get away from using classes).

the reason to use the name is for optimization purposes; it should speed up the search due to the way nokogiri indexes elements. since this is searching over a pretty small area, i don't expect the effects to be noticeable, but still good to do